src/Modules/OrchardCore.Commerce/OrchardCore.Commerce.csproj

```diff
@@ -26,7 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.0.0-alpha.occ-15.3" />
+    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.0.0" />
```

Comment:
Why is this reference needed here in the first place, @DAud-IcI?

Reply:
What do you mean exactly? We are using NodeJs here from the first Stripe issue, so we need it, because of the styling and js minification.

Reply:
Exactly, it lints our scripts.

Reply:
Magnificent ✨

I just saw the reference in the web project, too, and that none of the extra properties, that were defined there, were present here, so I figured I'd ask.